apps/client/src/widgets/mypage/components/account-menu-bar/account-menu-bar.tsx

```diff
@@ -0,0 +1,79 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+
+import { TextButton, useModal } from '@bds/ui';
+
+import LogoutModal from '@widgets/mypage/components/modal/logout-modal';
+import WithdrawModal from '@widgets/mypage/components/modal/withdraw-modal';
+
+import { USER_MUTATION_OPTIONS } from '@shared/api/domain/mypage/queries';
+import { USER_QUERY_KEY } from '@shared/api/keys/query-key';
+import { authService } from '@shared/auth/services/auth-service';
+import { routePath } from '@shared/router/path';
+
+import * as styles from './account-menu-bar.css';
+
+const AccountMenuBar = () => {
+  const queryClient = useQueryClient();
+
+  const { mutate: kakaoLogout } = useMutation({
+    ...USER_MUTATION_OPTIONS.KAKAO_LOGOUT(),
+    onSuccess: () => {
+      queryClient.invalidateQueries({
+        queryKey: USER_QUERY_KEY.ALL,
+      });
+
+      authService.logout();
+    },
+  });
+
+  const { mutate: kakaoWithdraw } = useMutation({
+    ...USER_MUTATION_OPTIONS.KAKAO_WITHDRAW(),
+    onSuccess: () => {
+      queryClient.invalidateQueries({
+        queryKey: USER_QUERY_KEY.ALL,
+      });
+
+      authService.logout();
+    },
+  });
+  const { openModal, closeModal } = useModal();
+
+  const handleLogout = () => {
+    openModal(
+      <LogoutModal
+        onLogout={() => {
+          closeModal();
+          kakaoLogout(window.location.origin + routePath.LOGIN);
+        }}
+        onCancelLogout={closeModal}
+      />,
+    );
+  };
+
+  const handleWithdraw = () => {
+    openModal(
+      <WithdrawModal
+        onWithdraw={() => {
+          closeModal();
+          kakaoWithdraw();
+        }}
+        onCancelWithdraw={closeModal}
+      />,
+    );
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.leftButtonWrapper}>
+        <TextButton color="black" size="sm" onClick={handleWithdraw}>
+          회원탈퇴
+        </TextButton>
+      </div>
+      <TextButton color="black" size="sm" onClick={handleLogout}>
+        로그아웃
+      </TextButton>
```

Comment:
```suggestion
      <TextButton color="black" size="sm" onClick={handleWithdraw}>
        회원탈퇴
      </TextButton>
      <div className={styles.verticalLine} />
      <TextButton color="black" size="sm" onClick={handleLogout}>
        로그아웃
      </TextButton>
```

저는 `<div className={styles.verticalLine} />` 
이렇게 세로 선이 있다는 사실이 보이는게 제 취향이라서 그냥 코멘트로 남겨봐요... (정말 개인 취향임)

Reply:
오 차라리 그게 더 낫겠네요! widgets에 세로 컴포넌트를 만들어야되나 생각했다가 그냥 wrapper로 감싸야지만 생각했었어요. styles로 하는게 더 직관적일것 같긴 하네요!